src/config/ConfigManager.hpp

```diff
@@ -156,9 +156,12 @@ class CConfigManager {
     void                                                            setInt(const std::string&, int);
     void                                                            setString(const std::string&, const std::string&);
 
-    int                                                             getDeviceInt(const std::string&, const std::string&, std::optional<bool> touchpad = {});
-    float                                                           getDeviceFloat(const std::string&, const std::string&, std::optional<bool> touchpad = {});
-    std::string                                                     getDeviceString(const std::string&, const std::string&, std::optional<bool> touchpad = {});
+    int                                                             getDeviceInt(const std::string&, const std::string&);
```

Comment:
makes no sense to dupe those. Just make fallback be default to "" and if it's `.empty()` then behave like it wasnt specified